README.md

```diff
@@ -49,6 +49,13 @@ sudo apt install make gcc openjdk-8-jdk pkg-config meson zip \
                  libsdl2-dev
 ```
 
+For Fedora you need to install the following packages:
```

Comment:
(nitpicking) Please add an empty line (for consistency) ;)

Reply:
added the new line